proto/cosmos/circuit/v1/tx.proto

```diff
@@ -44,10 +44,10 @@ message CircuitBreakerPermissions {
   // level is the level of permissions granted to this account.
   Level level = 1;
 
-  // limit_type_urls is used with LEVEL_SOME_MSGS to limit the lists of Msg type
-  // URLs that the account can pause. It is an error to use limit_type_urls with
+  // limit_msg_types is used with LEVEL_SOME_MSGS to limit the lists of Msg type
+  // name that the account can pause. It is an error to use limit_msg_types with
   // a level other than LEVEL_SOME_MSGS.
-  repeated string limit_type_urls = 2;
+  repeated string limit_msg_types = 2;
```

Comment:
This is a breaking change which requires the package version to be incremented from v1 to v2.

Reply:
This has never been released so no it isn't